src/components/app/Layout/Footer/index.jsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import cn from 'clsx'
+import { Link } from 'react-router-dom'
+import { menuFooter } from '../../../../libs/footer'
+import styles from './Footer.module.scss'
+import logo_footer from '../../../../assets/images/logo_footer.svg'
+import icon_vk from '../../../../assets/images/icon_social_vk.svg'
+import icon_twitter from '../../../../assets/images/icon_social_twitter.svg'
+import icon_facebook from '../../../../assets/images/icon_social_facebook.svg'
+
+const Footer = () => {
+  const wrapper = cn('wrapper', styles.footerWrapper)
+  return (
+    <div className={styles.footer}>
+      <div className={wrapper}>
+        <div className={styles.logoSocial}>
+          <img src={logo_footer} alt="logo" className={styles.logoFooter} />
+          <div className={styles.socialMedia}>
+            <img src={icon_vk} alt="logo" className={styles.iconVk} />
```

Comment:
наверное, здесь и ниже в alt лучше прописать название соцсети, наподобие
```suggestion
            <img src={icon_vk} alt="VK" className={styles.iconVk} />
```